recipes/pcapplusplus/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "PcapPlusPlus" %}
+{% set version = "22.05" %}
+{% set sha256 = "5f299c4503bf5d3c29f82b8d876a19be7dea29c2aadcb52f2f3b394846c21da9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/seladb/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

No need for indirection here.
```suggestion
  sha256: 5f299c4503bf5d3c29f82b8d876a19be7dea29c2aadcb52f2f3b394846c21da9
```